gen_mod_headers

```diff
@@ -246,16 +236,16 @@ if [[ -n "$prefix" ]]; then
 	mv .backup.Makefile.build scripts/Makefile.build
 	mv .backup.mod.Makefile scripts/mod/Makefile
 	mv .backup.kconfig.Makefile scripts/kconfig/Makefile
-	if [[ -e tools/build/Makefile ]]; then
-		mv .backup.tools.build.Makefile tools/build/Makefile
-	fi
 
 	# Clean up host script bins.
 	rm -rf xscripts
 	if [ -d xtools ]; then
 		rm -rf xtools
 	fi
 
+	# Remove unneeded binaries
+	rm -rf arch/x86/tools/relocs
+
```

Comment:
Can you please elaborate on why this is needed?

Reply:
Because now we only do `modules_prepare` which builds for the host machine. We used to do the `make HOSTCC=... scripts\` thingie which rebuilt all including this relocs binary for the target so nobody complained at the end because this `relocs` binary had the expected target interpreter.